packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -357,17 +361,33 @@ export class ChatWidget extends Component {
 
       let scrollButton;
       if (props.widget.showScrollToBottomButton) {
-        const label = widget.hasNewMessage ? `New Messages` : null;
+        const newMessages = {
+          id: `newMessages`,
+          defaultMessage: `New Messages`
+        };
+        const label = widget.hasNewMessage ? formatMessage(newMessages) : null;
         scrollButton = <ScrollToBottomButton label={label} onClick={this.handleScrollToBottom} />;
       }
 
       let deleteAlert;
       if (props.widget.showAlertModal) {
+        const confirmDeletingMessage = {
+          id: `confirmDeletingMessage`,
+          defaultMessage: `Are you sure you want to delete this message?`
+        };
+        const deleteMessage = {
+          id: `delete`,
+          defaultMessage: `Delete`
+        };
+        const cancelMessage = {
+          id: `cancel`,
+          defaultMessage: `Cancel`
+        };
```

Comment:
Nitpick: Do we need to declare these extra constants? I'm not sure that it really helps readability much.